### ISSUE TITLE
Dropdown with IconButton example has no accessible name

### ISSUE THREAD

**Issue by bradenmacdonald** [openedx/paragon#3616]
In this example: https://paragon-openedx.netlify.app/components/dropdown/#with-iconbutton

The dropdown button is missing an `aria-label` or equivalent and so has no accessible name. Non visual users will know it's a button and may be told it has a popup, but won't know what it's for.

![Image](https://github.com/user-attachments/assets/621284ae-0c88-46c5-81b7-7d21e848229f)

A simple fix is to add `aria-label="More actions"` to the `<Dropdown.Toggle>`, I think.
